mpt_extension_sdk/mpt_http/mpt.py

```diff
@@ -273,10 +274,18 @@ def get_product_onetime_items_by_ids(mpt_client, product_id, item_ids):
 
 
 @wrap_mpt_http_error
-def get_product_items_by_period(mpt_client, product_id, period):
+def get_product_items_by_period(
+    mpt_client,
+    product_id: str,
+    period: str,
+    filter_vendors: Iterable[str] | None = None,
```

Comment:
pls, rename it to vendor_external_ids or 'skus'. Otherwise it means nothing in our business field :-(

Reply:
Done